src/Umbraco.Infrastructure/HostedServices/RecurringHostedServiceBase.cs

```diff
@@ -71,7 +81,7 @@ public async void ExecuteAsync(object state)
             }
             catch (Exception ex)
             {
-                _logger.LogError(ex, "Unhandled exception in recurring hosted service {serviceName}.", GetType().Name);
+                _logger.LogError(ex, "Unhandled exception in recurring hosted service {serviceName}.");
```

Comment:
Noticed you removed the type parameter but missed the placeholder `{serviceName}` in the log message itself.

Reply:
@vsilvar good catch, we've toyed a bit around with different loggers so this was just an accidental removal 😅